### PR TITLE
KLC-162 Decode TX

### PR DESCRIPTION
**Pull request by fbsobreira**
Offline decode transaction from Raw to Human readable:
- Encode address into bech32
- Signature in hex format
- Data into UTF-8 string array

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -211,6 +211,9 @@ import { Transaction, Contracts, TXContract_ContractType, utils } from "@klever/
       "To Broadcast Format": tx.toBroadcast(),
       "Json Format": tx.toJSON(), 
     });
+
+  // decode TX will encode address in bech32 format and Uint8Array data into utf-8 string array
+  console.log(tx.decode())
```
Comment (Beroni) [inline README.md:216]: remove console.log

File: lib/transaction/contracts.ts [hunk 2]
```diff
@@ -0,0 +1,356 @@
...
+import * as proto from "./../../google/protobuf/any";
+import { 
+  Transaction as T,
+  TXContract_ContractType,
+} from './proto/transaction'
+
+
+const enc = new TextDecoder();
+
+const decodeTransferContract = (wired: Uint8Array): any => {
```
Comment (Beroni) [inline lib/transaction/contracts.ts:12]: don't use type `any`, create the correct type

Comment (fbsobreira) [reply lib/transaction/contracts.ts:12]: transaction cant by typed...

File: README.md (hunk 1 above)
Comment (fbsobreira) [reply README.md:216]: this is for illustration propus, readme file....

**Review by Beroni — APPROVED** [review]

**Comment by fbsobreira** [comment]
// todo: add type on decode TXs: c/c @Duukaa

**Pull request merged**